src/sentry/static/sentry/app/views/dashboardsV2/dashboard.tsx

```diff
@@ -1,7 +1,10 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
+// eslint-disable-next-line sentry/no-react-hooks
+import React, {useEffect} from 'react';
```

Comment:
shouldn't we wait until everything is in ts?

Reply:
This is being used in the same function as where `useSortable()` is used. Could an exception be made? I don't think it's worth it to create a new React class component to access the lifecycle methods.

Reply:
it's fine from my side, I'm looking forward to using hooks: D